ios/REANodesManager.m

```diff
@@ -505,6 +520,24 @@ - (void)updateProps:(nonnull NSDictionary *)props
       ofViewWithTag:(nonnull NSNumber *)viewTag
            withName:(nonnull NSString *)viewName
 {
+  ComponentUpdate *lastSnapshot = _componentUpdateBuffer[viewTag];
+  if (_viewRegistry[viewTag].superview == nil || lastSnapshot != nil) {
```

Comment:
It'd be good to maybe extract this to a method and call it sth like "isNativeViewFullyMounted" such that when doing this check it is easier to tell what exactly we are testing